segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/hdfs/HDFSStorage.java

```diff
@@ -136,52 +139,187 @@ public void initialize(long epoch) {
             conf.set("dfs.client.block.write.replace-datanode-on-failure.policy", "NEVER");
         }
 
-        this.context = new FileSystemOperation.OperationContext(epoch, openFileSystem(conf), this.config);
+        this.context = new OperationContext(epoch, openFileSystem(conf), this.config);
```

Comment:
Why do we still need `OperationContext`?

Reply:
We do not need this. Dropping the context.